openshift/templates/api/api-deploy.openvp.param

```diff
@@ -0,0 +1,33 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/api/api-deploy.json
+#=========================================================
+NAME=api
+APP_NAME=issuer
+APP_GROUP=openvp
+SUFFIX=-openvp
+ROLE=controller
+IMAGE_NAMESPACE=bztwou-tools
+SERVICE_PORT=8080
+DB_SERVICE=db
+DB_PORT=27017
+DB_NAME=issuer_db
+SOURCE_IMAGE_NAME=api
+TAG_NAME=dev
+AGENT_SERVICE_NAME=agent
+AGENT_ADMIN_PORT=8024
+SMTP_SERVER_ADDRESS=
+SMTP_SERVER_PORT=
+ADMIN_EMAIL=
+PUBLIC_SITE_URL=https://openvp-issuer-dev.pathfinder.gov.bc.ca/
+EXISTING_SCHEMA_ID=85459GxjNySJ8HwTTQ4vq7:2:verified_person:1.4.0
+APP_CONFIG_MAP_NAME=api-config
+CUSTOM_SCHEMA_PATH=
```

Comment:
Leaving this environment variable blank will make the api not look for the custom schema definition file. I suspect the deployment configurations rely on this to complete the mount path for the custom file, so they may need tweaking.